about.html

```diff
@@ -0,0 +1,127 @@
+      </div>
+    </section>
+    Program section
+    <section class="program" id="program">
+      <h1 class="main-program">Main Program</h1>
+        <hr class="line">
+      <div class="program-container">
+        generated dynamically
+      </div>
+      <button type="button" class="join-button" id="join">Be a Part of SCDK Program 2022</button>
+        <h3 class="see-program">SEE THE WHOLE PROGRAM</h3>
+    </section>
+    Speakers section -->
+    <!-- <section class="speakers">
+      <h1 class="header-speakers">Featured Speakers</h1>
+        <hr class="line">
+        <div class="speakers-container"> -->
+    <!--Dynamically Generated-->
+        <!-- </div>
+        <button type="button" class="more-button" id="more">MORE <img src="images/arrow-down.svg" width= 12px height= 12px alt="show more"> </button>
+    </section> -->
+    <section class="about" id="about">
+        <h1 class="headline-hello">GIS Expert world</h1>  
+        <h2 class="about-seminar">AFRICAN GIS EXPERT<br>INTERNATIONAL SEMINAR <br> 2022</h2>
+        <p class="about-para">A global network of professional GIS Practisioners 
+          aimed at sharing opportunities and creating a positive change 
+          with people from over 60 countries is holding in Nigeria.</p>
+        <p class="about-contact">Please contact us via Email for any further 
+          questions about SCDK GIS Seminar 2022!<br><span class="about-mail">scdkseminar2022@scdk.org</span>
+        </p>
+    </section>
+    <section class="seminar-logo">
+        <h1 class="seminar-logo-header">SCDK GIS Seminar 2022 Logo</h1>
+        <hr class="line">
+        <p class="seminar-logo-para">The logo of SCDK GIS Seminar 2022 was decided through the logo compettition from 10, November to 9, December, 2021.</p>
+        <p class="seminar-logo-para2">After online public voting and selection Committees Judging process, 
+            the logo from Ademola Adebayo won the prize.<br> After consultation with Tpl. Adebayo we optimized 
+            the prize-winning-logo for this year's summit as following:
+        </p>
+        <div class="Smart-logo">
+            <img src="images/logo3.png" alt="Smart-logo">
+        </div>
+    </section>
+    <section class="past-seminar" id="news">
+        <h1 class="past-seminar-header">See past SCDK GIS Seminars</h1>
+        <hr class="line">
+        <p class="past-seminar-para">Take a look at the last two SCDK GIS Seminars
+             which took place in Ibadan.
+        </p>
+        <div class="past-seminars-images">
+            <img src="images/conference1.JPG" alt="past seminar">
+            <img src="images/conference2.JPG" alt="past seminar">
+        </div>
+    </section>
+    <section class="partner">
+        <h1 class="main-program">Partner</h1>
+        <hr class="line">
+        <div class="partner-list">
+            <img src="images/IUFMP.png" alt="IUFMP">
+            <img src="images/UIlogo.png" alt="UI">
```

Comment:
- **Optional**
The partner section doesn't exist in the original design, therefore I recommend removing it to fully commit to the original project.